common/src/main/java/io/homeassistant/companion/android/common/sensors/NfcSensorManager.kt

```diff
@@ -0,0 +1,51 @@
+package io.homeassistant.companion.android.common.sensors
+
+import android.content.Context
+import android.nfc.NfcAdapter
+import io.homeassistant.companion.android.common.R as commonR
+
+class NfcSensorManager : SensorManager {
+    companion object {
+        private const val TAG = "NfcSensor"
+
+        val nfcStateSensor = SensorManager.BasicSensor(
+            "nfc_state",
+            "binary_sensor",
+            commonR.string.sensor_name_nfc_sensor,
+            commonR.string.sensor_description_nfc_sensor,
+            "mdi:nfc",
```

Comment:
It may depend on the device but for me the system settings icon for NFC is actually `mdi:nfc-variant` (on a Google Pixel device)

Reply:
Done!